subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/mqtt/MqttNotificationService.kt

```diff
@@ -0,0 +1,120 @@
+package com.egm.stellio.subscription.service.mqtt
+
+import com.egm.stellio.shared.model.BadSchemeException
+import com.egm.stellio.shared.util.JsonUtils.serializeObject
+import com.egm.stellio.subscription.model.Notification
+import com.egm.stellio.subscription.model.Subscription
+import org.eclipse.paho.client.mqttv3.MqttClient
+import org.eclipse.paho.client.mqttv3.MqttConnectOptions
+import org.eclipse.paho.client.mqttv3.MqttMessage
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence
+import org.eclipse.paho.mqttv5.client.IMqttToken
+import org.eclipse.paho.mqttv5.client.MqttAsyncClient
+import org.eclipse.paho.mqttv5.client.MqttConnectionOptions
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.eclipse.paho.client.mqttv3.MqttException as MqttExceptionV3
+import org.eclipse.paho.mqttv5.common.MqttException as MqttExceptionV5
+
+@Service
+class MqttNotificationService(
+    @Value("\${subscription.mqtt.clientId}")
+    private val clientId: String = "stellio-context-broker",
```

Comment:
there is already a (different) default value for `subscription.mqtt.clientId` in application.properties. I suggest to remove the one here.